src/gui/plugins/view_angle/ViewAngle.qml

```diff
@@ -182,5 +183,130 @@ ToolBar {
         ViewAngle.OnAngleMode(0, 0, 1)
       }
     }
+
+    // set camera pose
+    // xyz
+    Text {
+      Layout.columnSpan: 2
+      Layout.row: 0
+      Layout.column: 5
+      color: "dimgrey"
+      font.bold: true
+      text: "Position (m)"
+    }
+
+    Text {
+      text: "X"
+      color: "dimgrey"
+      Layout.row: 1
+      Layout.column: 5
+    }
+    IgnSpinBox {
+      id: x
+      Layout.row: 1
+      Layout.column: 6
+      value: ViewAngle.camPose[0]
+      maximumValue: 1000.00
+      minimumValue: -1000.00
```

Comment:
I think we can increase these limits. 1 km can be too small for large worlds

Reply:
How's 1000km? 0cdb38b3fdf9d3bebd4c2cc4a3d362103453de07